src/frontend/overlay/src/utils/hooks/withTimeoutAfterRender.js

```diff
@@ -0,0 +1,12 @@
+import { useEffect, useState } from "react";
+
+export const useWithTimeoutAfterRender = (timeout) => {
```

Comment:
just useDelayed
Also may be a good idea to add a second parameter - the value that should be set.
Might work like that:
first render - returns undefined
time passed - returns value passed as argument